notifications_utils/recipients.py

```diff
@@ -520,6 +521,17 @@ def validate_and_format_email_address(email_address):
 
 
 def validate_address(address_line, column):
+    if address_line:
+        def get_invalid_chars(s):
+            invalid_chars = [c for c in s if ord(c) > 255]
+            return sorted(set(invalid_chars))
+
+        invalid_chars = get_invalid_chars(address_line)
+        if len(invalid_chars) > 0:
+            raise InvalidAddressError(
+                "Can`t include: {}".format(
```

Comment:
Two things:
- can you remove the colon
- typographic apostrophe looks like `’`